src/main/java/org/opensearch/ad/transport/GetAnomalyDetectorRequest.java

```diff
@@ -42,7 +43,7 @@
     private String typeStr;
     private String rawPath;
     private boolean all;
-    private String entityValue;
+    private Entity entityValue;
```

Comment:
nit - should this var be renamed to `entity` instead of `entityValue`, since it now includes a list of attributes rather than a single value?

Reply:
right, changed